docs/source/reference/modding/extended.rst

```diff
@@ -0,0 +1,212 @@
+because they have their own walking animations, they override ones which come from ``xbase_anim`` and its loose overrides.
+
+To enable this feature, you should have this line in your settings.cfg:
+
+::
+
+    [Game]
+    use additional anim sources = true
+
+An example of mod which uses this feature is `Almalexia's Cast for Beasts`_.
+
+
+Weapon sheathing support
+------------------------
+
+In OpenMW it is possible to display equipped, but not currently wielded weapons on the actor's model, including quivers and scabbards.
+
+This feature conflicts with old mods which use scripted scabbards, arrows with particles or decorative quivers (attached to the left pauldron, for example).
+
+1. Basics
+
+The minimum you need is the ``xbase_anim_sh.nif`` file from the `Weapon Sheathing`_ mod and this line in your settings.cfg:
+
+::
+
+    [Game]
+    weapon sheathing = true
+
+The ``xbase_anim_sh.nif`` contains default placement points for different weapon types.
+That way you'll get Gothic-style weapon sheathing for all biped actors (without quivers and scabbards).
+
+2. Scabbards
+
+For a scabbard to be displayed, you need a mesh with ``_sh`` suffix. For example, if the weapon has a model named foo.nif, the scabbard model must be named foo_sh.nif.
+
+There should be an least two nodes in the sheath file:
+
+``Bip01 Weapon`` - represents the weapon itself (may be just a grip for sword, for example). Is not shown when the weapon is drawn.
+
+``Bip01 Sheath`` - represents scabbards, quivers, etc. It is shown always when the weapon is equipped.
+
+You can move or rotate nodes if the default placement from the ``xbase_anim_sh.nif`` does not look good for your weapon.
+
+If you want to exempt a specific weapon from using this feature, you can create a stub sheath mesh with just one root node.
+
+If you want to use the common weapon mesh, but with custom placement, you can create a sheath mesh with an empty ``Bip01 Weapon`` node and move it as you want.
+
+3. Quivers
+
+To show the quiver for a ranged weapon, you need these nodes in the sheath file:
+
+``Bip01 Sheath`` node, as for scabbards
+
+``Bip01 Ammo`` node to show ammunition in the quiver
+
+``Bip01 Weapon`` to show the weapon itself (not needed for throwing weapons)
+
+The ``Bip01 Ammo`` should have some empty child nodes, to which the engine will attach ammunition nodes.
+
+The appearance and count of shown ammunition depends on type and count of equipped ammunition. It the ammunition has a wrong type (e.g. bolts for bow), it won't be shown.
+
+It is important to make sure the names of empty nodes start with ``"Bip01 "``, or the engine will optimize them out.
+
+4. Shields
+
+Shields holstering is not supported at the moment since it conflicts with any mods which use pseudo-shields as held items (such as Animated Morrowind and Hold It).
```

Comment:
```suggestion
Shield holstering is not supported at the moment since it conflicts with any mods which use pseudo-shields as held items (such as Animated Morrowind and Hold It).
```